history.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import unicode_literals
+from ably import AblyRest
+
+client = AblyRest('{{ApiKey}}')
```

Comment:
This is missing the friendly error message for users who run this demo without setting the API Key.  It is important that the user is given an error message that makes it obvious they need to add the API to run the demo